targettests/Remote-Sim/get_state.cpp

```diff
@@ -27,6 +27,7 @@ int main() {
     assert(std::abs(state[2].real()) < 1e-3);
     assert(std::abs(M_SQRT1_2 - state[3].real()) < 1e-3);
   }
+
 // Skipped test due to a stability issue. See:
 // https://github.com/NVIDIA/cuda-quantum/issues/1087
 #if 0
```

Comment:
nit: delete this block?

Reply:
We'll keep this block since I haven't had a chance to retest it according to the ticket linked. We probably want to re-enable the test if it's no longer unstable.